src/opers.h

```diff
@@ -16,7 +16,7 @@
 
 #include "system.h"
 #include "calls.h"
-
+#include "bool.h"
```

Comment:
Please keep #includes sorted.

But in this case, I'd rather avoid the include; see below for more on this.